src/main/java/seedu/address/logic/parser/PermaSortContactCommandParser.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.exceptions.IllegalValueException;
+import seedu.address.logic.commands.PermaSortContactCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+/**
+ * Parses input arguments and creates a new {@code SortCommand} object
+ */
+public class PermaSortContactCommandParser implements Parser<PermaSortContactCommand> {
+    /**9
```

Comment:
There is an additional '9' here. Also consider leaving a line between the class declaration and the first javadocs